android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -398,6 +417,51 @@ public void run() {
         }, 1);
     }
 
+    private DrmSessionManager<FrameworkMediaCrypto> buildDrmSessionManager(UUID drmUUID, Uri streamUrl, String licenseServerUrl, String drmToken) {
+        if (drmUUID == null) {
+            return null;
+        }
+        if (Util.SDK_INT < 18) {
+            Log.d(TAG, "Unsupported DRM scheme " + drmUUID.toString());
+            eventEmitter.error(getResources().getString(R.string.error_drm_not_supported), new UnsupportedDrmException(UnsupportedDrmException.REASON_UNSUPPORTED_SCHEME));
+            return null;
+        }
+        if (drmUUID != vudrm.widevineDRMSchemeUUID) {
+            Log.d(TAG, "Unsupported DRM scheme " + drmUUID.toString());
+            eventEmitter.error(getResources().getString(R.string.error_drm_unsupported_scheme), new UnsupportedDrmException(UnsupportedDrmException.REASON_UNSUPPORTED_SCHEME));
+            return null;
+        }
+        AssetConfiguration assetConfiguration = null;
+        try {
+            assetConfiguration = new AssetConfiguration.Builder()
+                    .tokenWith(drmToken)
+                    .licenceUrlWith(licenseServerUrl)
+                    .kidProviderWith(
+                            new HttpKidSource(
+                                    new URL(streamUrl.toString())
+                            )
+                    )
+                    .build();
```

Comment:
Indentation :thinking:

Reply:
I am just sticking to the automatic format.